server/actions/compute-handler.js

```diff
@@ -19,9 +19,9 @@ function computeHandler(req, res) {
     const startTime = new Date();
     compute(user, ship, { preview: true })
     .then(result => {
-      const logs = result.logs;
-      if (logs && logs.length) {
-        logs.map(line => req.hull.client.logger.debug("preview.console.log", line));
+      const logsForLogger = result.logsForLogger;
+      if (logsForLogger && logsForLogger.length) {
+        logsForLogger.map(line => req.hull.client.logger.debug("preview.console.log", line));
```

Comment:
@mkawalek just to verify -> in UI preview we want to show both console.log and console.info, is it the case?